dist/models/DataManager.js

```diff
@@ -0,0 +1,46 @@
+class DataManager {
+  constructor() {
+    this.data = {
+      currentTeam: [],
+      dreamTeam: [],
+    };
+  }
+
+  loadTeamPlayers = (teamName) => {
+    $.get(`/teams/${teamName}`, (res) => {
+      this.data.currentTeam = res;
+    });
+  };
+
+  loadPlayerStats = (playerId) => {
+    const renderer = new Renderer();
+    const player = this.data.currentTeam.find((p) => p.id === playerId);
+    $.get(`/player/${player.lastName}/${player.firstName}`, (res) => {
+      player.stats = res;
+      renderer.renderPlayerStats(player.stats, playerId);
```

Comment:
instead of rendering here, you should have return the data to main.js (where this function is invoked) and then in main.js use the renderer with that data.
not very good practice to create another instance of the render here in the DataManager...